.travis.yml

```diff
@@ -100,7 +109,8 @@ script:
   - |
     if [[ $BUILD_DOCS == false ]]; then
       export MPL_REPO_DIR=$PWD  # needed for pep8-conformance test of the examples
-      gdb -return-child-result -batch -ex r -ex bt --args python tests.py --processes=$NPROC --process-timeout=300 $TEST_ARGS
+      # gdb -return-child-result -batch -ex r -ex bt --args python tests.py --processes=$NPROC --process-timeout=300 $TEST_ARGS
+      py.test --pep8 --cov=matplotlib ./lib/matplotlib/tests
```

Comment:
In reference to your question about the `matplotlib._path` C++ extension not loading...

Here, you are running the tests from the source directory, not the installed directory (which usually ends up under site-packages).  I think if you give `py.test` an import path, rather than a file path, it should import matplotlib from the installed directory, i.e.:

```
py.test matplotlib.tests
```